possel/application.py

```diff
@@ -110,26 +112,31 @@ def get_arg_parser():
 def main():
     args = get_arg_parser().parse_args()
```

Comment:
Settings is an inappropriate value here? That's specifically tornado-only information. !fix